.github/workflows/ci.yaml

```diff
@@ -33,17 +44,24 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-go-
 
-      - name: Install tools
-        run: ./setup.sh
+      - uses: chainguard-dev/actions/goimports@main
 
-      - name: Check License Headers
-        run: mage -v checkLicenseHeaders
-
-      - name: Lint
-        run: mage -v lint
-
-      - name: Test
-        run: mage -v test
+      - name: Run Mage
+        uses: magefile/mage-action@v2
+        with:
+          version: v1.13.0
+          args: -v ci
 
-      - name: Build
-        run: mage -v build
+  golangci:
+    name: lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-go@v3
+        with:
+          go-version: '1.18'
```

Comment:
BTW, this can point to the mod-file and select version automagically based on that

Reply:
done, thanks @davidkarlsen PTAL